src/PhpStan/Main.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Kotamat\WebdbSandbox\PhpStan;
+
+class Main
+{
+    public function error()
+    {
+        // 存在しないクラスなのでエラー
+        new ErrorClass();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Instantiated class Kotamat\WebdbSandbox\PhpStan\ErrorClass not found.